File: solution/2706. Buy Two Chocolates/2706.cpp
class Solution {
public:
    int buyChoco(vector<int>& prices, int money) {
        int mn1 = 101, mn2 = 101;
        for (int x : prices) {
            if (x < mn1) {
                mn2 = mn1;
                mn1 = x;
            } else if (x < mn2) {
                mn2 = x;
            }
        }
        return mn1 + mn2 > money ? money : money - mn1 - mn2;
    }
};

File: solution/2706. Buy Two Chocolates/2706.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        mn1, mn2 = 101, 101
        
        for x in prices:
            if x < mn1:
                mn1, mn2 = x, mn1
            elif x < mn2:
                mn2 = x
        
        return money - mn1 - mn2 if money >= mn1 + mn2 else money
    
